assets/css/_location_card.scss

```diff
@@ -0,0 +1,47 @@
+.c-location-card.c-location-card--selection {
+  width: calc(100% - 2rem);
+
+  .c-card__icon {
+    svg {
+      width: 1.6875rem !important;
+      height: 2.25rem !important;
+    }
+  }
+}
+
+.c-location-card {
+  padding: 1rem;
+
+  .c-card__top-row {
+    margin-top: 0;
+
+    &:empty {
+      min-height: 0;
+    }
+  }
+
+  .c-card__contents {
+    margin-top: 0.19rem;
+
+    &:empty {
+      margin-top: 0;
+    }
+
+    .c-street-view-button {
+      margin-top: 0.5rem;
+    }
+  }
+
+  .c-card__icon {
+    padding-right: 0.5rem;
+
+    svg {
+      width: 1.125rem !important;
+      height: 1.5rem !important;
+
+      path {
+        fill: #ea5382;
```

Comment:
suggestion (non-blocking): Consider referencing `$color-strawberry-500`

Reply:
Ah I totally missed that this was a preexisting color, thanks.